### PR TITLE
Page bicicleta interna

### PR DESCRIPTION
**Pull request by taras-zherebetskyi**

### CODE REVIEW

File: src/components/Сhoice-of-bikes/ChoiceBikes.jsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import quantity from "./img/quantity.png";
+
+import dec from "../img/dec.svg";
+
+const bike1 = "Nimbus Stack";
+const bike2 = "Magic Migth";
+const bike3 = "Mebula Cosmic";
+
+export const BicicletaInterna = ({ img, title}) => {
+  let isMain = img ? "main" : "small";
```
Comment (taras-zherebetskyi) [inline src/components/Сhoice-of-bikes/ChoiceBikes.jsx]: Why u use let here?

change variable name
is____ that mean boolean type

File: src/components/Сhoice-of-bikes/choiceBikes.scss [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      }
+      &-subtitle{
+        margin-top: 5px;
+      }
+    }
+  }
+}
+#bike-choice-quantity {
+  margin-top: 16px;
+}
```
Comment (taras-zherebetskyi) [inline src/components/Сhoice-of-bikes/choiceBikes.scss]: Why are u applying for id?

File: src/components/Сhoice-of-bikes/ChoiceBikes.jsx (hunk 1 above)
Comment (NazariiSova) [reply src/components/Сhoice-of-bikes/ChoiceBikes.jsx]: done

File: src/components/Сhoice-of-bikes/choiceBikes.scss (hunk 2 above)
Comment (NazariiSova) [reply src/components/Сhoice-of-bikes/choiceBikes.scss]: done

File: src/components/OthersBikes/OthersBikes.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./OthersBikes.scss";
+
+
+
+export const OtherBikes = ({ subBikes, handlerChangeItem }) => { 
+  console.log(subBikes);
```
Comment (taras-zherebetskyi) [inline src/components/OthersBikes/OthersBikes.jsx:7]: remove logs

File: src/pages/Bicicleta-interna/Bicicleta-interna.page.jsx [hunk 4]
```diff
@@ -1,10 +1,96 @@
...
+  const bikeId = searchParams.get("id");
+  const [mainBike, setMainBike] = useState(dataBikeCatalog[bikeId-1]);
+  const [subBikes, setSubBikes] = useState(
+    dataBikeCatalog.filter((bike) => {
+      return bike.title !== dataBikeCatalog[bikeId-1].title;
+    })
+  );
+
+  const handlerChangeItem = (event) => {
+    console.log(event);
```
Comment (taras-zherebetskyi) [inline src/pages/Bicicleta-interna/Bicicleta-interna.page.jsx:30]: remove log

File: src/pages/Home/Home.page.jsx [hunk 5]
```diff
@@ -48,21 +53,47 @@ const HomePage = () => {
...
+            >
+              <img
+                className="catalog-card-img"
+                src={bicicleta1}
+                alt="bicicleta1"
+                style={{ cursor: "pointer" }}
+              />
+            </Link >
             <div className="catalog-card-description">Nimbus Stark</div>
             <div className="catalog-card-price">R$ 4909</div>
```
Comment (taras-zherebetskyi) [inline src/pages/Home/Home.page.jsx:83]: can u create array and map it?

File: src/pages/Seguros/Seguros.page.jsx [hunk 6]
```diff
@@ -1,22 +1,160 @@
...
+                <div className="seguro-advantages-item-title">
+                  Reparo de Quadro
+                </div>
+                <div className="seguro-advantages-item-description">
+                  Nossos quadros são feitos para durar para sempre. Mas caso
+                  algo ocorra, ficamos felizes em reparar.
+                </div>
+              </div>
+              <div className="seguro-advantages-item-box">
+                <img
```
Comment (taras-zherebetskyi) [inline src/pages/Seguros/Seguros.page.jsx:92]: can u create array & map it here?

**Pull request merged**